src/content/docs/en/recipes/i18n.mdx

```diff
@@ -0,0 +1,322 @@
+---
+title: Add internationalization features 
```

Comment:
```suggestion
title: Add internationalization (i18n) features 
```
Was just looking at this on the site and my eye always looks for "i18n" specifically

Reply:
Good thinking. Also just confirmed this is much more frequently what people search for. I’m going to switch this up so we do include “internationalization” in the description.